pkg/cluster/reconcile.go

```diff
@@ -162,6 +156,8 @@ func (c *Cluster) resize() error {
 }
 
 func (c *Cluster) addOneMember() error {
+	c.status.AppendScalingUpCondition()
```

Comment:
If we call AppendScalingUpCondition() in addOneMember(), we also need to do the same in addOneSelfHostedMember().